content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -514,5 +514,5 @@ When you enable the API Priority and Fairness feature, the kube-apiserver serves
 For background information on design details for API priority and fairness, see
 the [enhancement proposal](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md).
 You can make suggestions and feature requests via [SIG API
-Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery).
+Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery) or our [slack channel](http://kubernetes.slack.com/messages/api-priority-and-fairness).
```

Comment:
nit:
```suggestion
Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery) or the SIG's
[Slack channel](http://kubernetes.slack.com/messages/api-priority-and-fairness).
```

Reply:
we dont have a dedicated sig yet. the feature is majorly maintained by api-machinery but also involves effort from scheduling and scalability. does that make sense?